libvirt/tests/src/virtual_network/qemu/hotplug_nic_with_repetition.py

```
@@ -0,0 +1,271 @@
+        # 3. hotplug a device
+        hotplug_mac, hotplug_type = attach_hotplug_interface(test, vm, original_iface)
+
+        # 4. check connection
+        session = vm.wait_for_serial_login(timeout=login_timeout)
+        test.log.info("First time to obtain hotplug nic ip address...")
+        hotnic_ip = get_hotplug_nic_ip(
+            test, vm, hotplug_mac, session, guest_is_linux, params
+        )
+
+        if not hotnic_ip:
+            test.log.info("First time failed，try to renew ip address...")
```

Ruff (0.14.10)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)